Common/cpp/NativeModules/NativeReanimatedModule.cpp

```diff
@@ -388,7 +388,7 @@ jsi::Value NativeReanimatedModule::configureLayoutAnimation(
     const jsi::Value &config) {
   layoutAnimationsManager_.configureAnimation(
       viewTag.asNumber(),
-      type.asString(rt).utf8(rt),
+      (LayoutAnimationType)type.asNumber(),
```

Comment:
C-style casts are not welcome, please use `static_cast` or define a new function `parseLayoutAnimationType` that accepts `const jsi::Value &`.